extension/ts/adapters/storage.ts

```diff
@@ -0,0 +1,19 @@
+import {makeCache} from "../util/cache";
+
+/**
+ * This file works across background and foreground threads. Consider migrating all storage to this
```

Comment:
is this being used right now or is it just POC?

Reply:
Yeah I already deleted all the code that used this lol